sdk/translation/Azure.AI.Translation.Document/tests/samples/Sample_PollIndividualDocuments.cs

```diff
@@ -10,20 +10,28 @@
 namespace Azure.AI.Translation.Document.Samples
 {
     [LiveOnly]
-    public partial class DocumentTranslationSamples : SamplesBase<DocumentTranslationTestEnvironment>
+    public partial class DocumentTranslationSamples : DocumentTranslationLiveTestBase
     {
         [Test]
-        [Ignore("Samples not working yet")]
+        [SyncOnly]
         public void PollIndividualDocuments()
         {
+#if SNIPPET
+            string endpoint = "<endpoint>";
```

Comment:
nit: in some places it says `"<Document Translator Resource Endpoint>";`
consider using the same message across the samples for consistency